### PR TITLE
Add comments

### PR DESCRIPTION
**Pull request by canyon289**
Added comments. Feel free to merge directly and review there, or do whatver is most useful to you.

There is no difference between red and green text. Just ignore the colors and read the words!

### CODE REVIEW

File: lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb [hunk 1]
```diff
@@ -2996,7 +2996,10 @@
    "cell_type": "markdown",
```
Comment (tomicapretto) [inline lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb:2996]: <p>I agree it makes sense to leave the section with that positive feeling. I'm moving this to the next section, but not all to the prior sub-section. I think leading questions about priors are good in the very beginning. Then I will mention that Bambi models contain many things, and, among them, the priors. </p><p> </p><p>Thanks for the idea! :D </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (tomicapretto) [inline lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb:2996]: <p>love it!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='99'/>

Comment (tomicapretto) [inline lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb:2996]: <p>It is an important distinction, but we don't have any practical examples at the moment. I would definitely write it down as something we need to address in the course, but I'm not sure if this is the right moment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='133'/>

Comment (tomicapretto) [inline lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb:2996]: <p>This is a great idea!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='229'/>

**Comment by tomicapretto** [comment]
@canyon289 I incorporated the changes in my branch. Feel free to close the PR once you've seen this comment.

Thanks a lot for the great ideas!

**Comment by canyon289** [comment]
ty

**Pull request closed without merging**